Orbitersdk/samples/ProjectApollo/src_sys/apolloguidance.cpp

```diff
@@ -463,6 +465,10 @@ void ApolloGuidance::LoadState(FILEHANDLE scn)
 	// Now load the data.
 	//
 
+	boolean  //Flag to keep compatibility with older scenario files.
```

Comment:
Put this on a single line like `boolean scalerLoaded = false; // Flag to keep compatibility with older scenario files.`

Reply:
Yeah there will more "something is loaded" variables will arrive here. That's why the multiline declaration.